src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml

```diff
@@ -1002,6 +1002,19 @@
 			<StaticResource x:Key="AccentButtonBorderBrushPointerOver" ResourceKey="SystemControlHighlightBaseMediumLowBrush" />
 			<StaticResource x:Key="AccentButtonBorderBrushPressed" ResourceKey="SystemControlHighlightTransparentBrush" />
 			<StaticResource x:Key="AccentButtonBorderBrushDisabled" ResourceKey="SystemControlDisabledTransparentBrush" />
+			
+			<!-- Light Dismiss Overlay Resources -->
+			<!--<StaticResource x:Key="AppBarLightDismissOverlayBackground" ResourceKey="SystemControlPageBackgroundMediumAltMediumBrush" />-->
```

Comment:
Why some of them are commented?

Reply:
Those ones are already present in the separate AppBar.xaml and CommandBar.xaml files. I left them there for easier tracking of upstream changes.